app/views/pages/events.html.erb

```diff
@@ -1,68 +1,77 @@
-      <th class="sort" data-sort="location">Location</th>
-      <th class="sort" data-sort="event_name">Event</th>
-      <th class="sort" data-sort="date">Date</th>
-    </tr>
-    </thead>
-    <tbody class="list">
-      <% @events.map do |e| %>
-        <% if e.current? %>
-          <tr>
-        <% else %>
-          <tr class="past">
+    <table class="table is-narrow is-narrow is-fullwidth event-table is-fluid">
+      <thead class="thead-sort">
+      <tr class= "sort-container">
+        <th class="sort" data-sort="location">Organizer's Location</th>
+        <th class="sort" data-sort="event_name">Event</th>
+        <th class="sort" data-sort="date">Date</th>
+        <th data-sort="date">RVSP</th>
+      </tr>
+      </thead>
+      <tbody class="list">
+        <% @events.map do |e| %>
+          <% if e.current? %>
+            <tr>
+          <% else %>
+            <tr class="past">
+          <% end %>
+              <td class="multisort is-hidden">
+                <%= e.current? ? 'Future' : 'Past'%>
+                <%= e.country %>
+                <%= e.date.strftime("%m %d") %>
+              </td>
+              <td class="location" data-label="location"><%= e.location %></td>
+              <td class="event_name" data-label="event">
+                <a class="emphasis" href="<%= e.url %>" target="blank" rel="noopener"><%= e.name %></a>
+                <br/>
+              </td>
+              <td class="date is-hidden"><%= e.date %></td>
+              <td data-label="date"><%= e.date.strftime("%Y/%m/%d") %></td>
+              <td><a class="rvsp" href="<%= e.url %>">RVSP here</a></td>
+            </tr>
         <% end %>
-            <td class="multisort is-hidden">
-              <%= e.current? ? 'Future' : 'Past'%>
-              <%= e.country %>
-              <%= e.date.strftime("%m %d") %>
-            </td>
-            <td class="location" data-label="location"><%= e.location %></td>
-            <td class="event_name" data-label="event">
-              <a class="emphasis" href="<%= e.url %>" target="blank" rel="noopener"><%= e.name %></a>
-              <br/>
-              
-            </td>
-            <td class="date is-hidden"><%= e.date %></td>
-            <td data-label="date"><%= e.date.strftime("%b %e") %></td>
-          </tr>
-      <% end %>
-    </table>
-  </div>
-  <p>
-    New events will be added throughout Hacktoberfest – please check back soon!
-  </p>
+      </table>
+    </div>
+    <p>
+      New events will be added throughout Hacktoberfest – please check back soon!
+    </p>
+
+    <section>
+      <br/>
+      <br/>
+      <div class="line-br">
+        <span class="line-br-1">-</span>
+        <span class="line-br-2">-</span>
+      </div>
+      <br/>
+      <h2 class="title is-2">List your event</h2>
+      <p>
+        To add your Hacktoberfest 2019 event to this page, please <%= link_to 'share the event details here', event_kit_path + "/#form" %>.
+      </p>
+    </section>
```

Comment:
Is the intention to be matching the static site for this? If so, should this copy be removed and replaced with the button at the top?

Reply:
I feel like this is left over copy from last year. This isn't in the new designs so I will remove it.